src/Core/Translation/Builder/Map/Message.php

```diff
@@ -52,6 +52,11 @@ class Message
      */
     private $userTranslation;
 
+    /**
+     * @var array
+     */
+    private $metadata;
```

Comment:
```suggestion
    private $metadata = [];
```

Otherwise, getMetadata can returns null :thinking: